Tests/ResultFormatSpec.swift

```diff
@@ -45,6 +45,34 @@ class ResultFormatSpec: QuickSpec {
                     fail("\(error)")
                 }
             }
+            
+            it("uses the custom date formatters") {
```

Comment:
Remove this test as its redundant. The previous test already ensures the date formatter is used by way of the `dateDecodingStrategy` being used.

Reply:
I disagree. This tests a completely different type of `dateDecodingStrategy` than the test above it. So I think this test should be kept. It doesn't hurt to have more tests. 👍

Reply:
You've already written a test which ensures the specified strategy is used. That you're simply using a different DateFormatter is moot, as the original test already uses a custom DateFormatter. This is a duplicate test. Please remove.

> It doesn't hurt to have more tests

It does when the test is a duplicate. It's redundant code, and I don't wish to maintain unnecessary code.

Reply:
I've removed it.